nav2_bt_navigator/README.md

```diff
@@ -18,85 +18,70 @@ BtNavigator:
 
 Using the XML filename as a parameter makes it easy to change or extend the logic used for navigation. Once can simply update the XML description for the BT and the BtNavigator task server will use the new description.
 
+## Behavior Tree nodes
+
+A Behavior Tree consists of control flow nodes, such as fallback, sequence, parallel, and decorator, as well as two execution nodes: condition and action nodes. Execution nodes are the leaf nodes of the tree. When a leaf node is ticked, the node does some work and it returns either SUCCESS, FAILURE or RUNNING.  The current Navigation2 software implements a few custom nodes, including Conditions and Actions. The user can also define and register additional node types that can then be used in BTs and the corresponding XML descriptions.
+
 ## Navigation Behavior Trees
 
-The BT Navigator package has a few sample XML-based descriptions of BTs.
+The BT Navigator package has three sample XML-based descriptions of BTs.  These trees are [navigate_w_replanning.xml](behavior_trees/navigate_w_replanning.xml), [navigate_w_replanning_and_recovery.xml](behavior_trees/navigate_w_replanning_and_recovery.xml), and [auto_localization_w_replanning_and_recovery.xml](behavior_trees/auto_localization_w_replanning_and_recovery.xml).  The user may use any of these sample trees or develop a more complex tree which could better suit the user needs.
```

Comment:
It should be either "could better suit the user's needs" or "could better suit user needs." The first one means a specific user's needs, while the second is referring to users in general. Given the first part of the sentence, I recommend the first option.